pkg/k8sutil/clientset.go

```diff
@@ -134,3 +138,17 @@ func GetDynamicResourceInterface(gvk *schema.GroupVersionKind, namespace string)
 
 	return dr, nil
 }
+
+func GetKubeClient(ctx context.Context) (kbclient.Client, error) {
```

Comment:
Are we able to use the existing `GetClientset` function we have as long as the EC scheme is registered?